index.html

```diff
@@ -1,11 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
-    <h1>Testing</h1>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header>
+        <h1 id="title">Survey Form</h1>
+    </header>
+    <main class="container">
+        <p id="description">Let us know how we can improve freeCodeCamp</p>
+        <form id="survey-form" action="index.html" method="post">
+            <fieldset>
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="name-label">Name: *</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="text" id="name" placeholder="Enter your name" required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="email-label">* Email:</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="email" id="email" placeholder="Enter your Email" required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="number-label">* Age:</label>
+                    </div>
```

Comment:
# Form label

- I feel the use of semantic tags like `fieldset` instead of generic tags like `<div>` for sectioning your form contents might improve the overall structure of your code and also improve accessibility. Kindly see [how to build a form in HTML](https://developer.mozilla.org/en-US/docs/Learn/Forms/Your_first_form) for insights

- Why don't you place the asterisks (*) within #email-label  and #number-label elements in front of the text instead of putting them behind the text, e,g. `<label id="email-label>Email *:</label>`. Placing the asterisk(*) after the text will greatly improve readability and accessibility.